3rdparty/include/webgpu/README.md

```diff
@@ -0,0 +1,30 @@
+## Build dawn to enable native implementation for DNN_BACKEND_WEBGPU  in modules/dnn
+
+
+### Build dawn and set the environment variable
+
+ Refer to [Dawn's build instructions](https://dawn.googlesource.com/dawn/+/HEAD/docs/buiding.md)  to complete the build of Dawn. Set environment variable `WEBGPU_ROOT_DIR` to enable native DNN_BACKEND_WEBGPU build: `export WEBGPU_ROOT_DIR=${PATH_TO_Dawn}`.
+
+### Test native DNN_BACKEND_WEBGPU backend
+Add -DWITH_WEBGPU=ON to the cmake command to build the webgpu module such as:
+`cmake -D CMAKE_BUILD_TYPE=Release -DWITH_WEBGPU=ON -D CMAKE_INSTALL_PREFIX=/usr/local ..`
+
+Temporarily following these instructions to do the test:
+```
+git clone https://github.com/opencv/opencv_extra.git
+export OPENCV_TEST_DATA_PATH=${PATH_TO}/opencv_extra/testdata
+
+sudo apt-get install libvulkan1 mesa-vulkan-drivers vulkan-utils
+git clone https://github.com/NALLEIN/opencv.git
+cd opencv/
+git checkout -b DawnTest origin/gsoc_2020_webgpu
+mkdir build
+cd build
+cmake -D CMAKE_BUILD_TYPE=Release -DWITH_WEBGPU=ON -D CMAKE_INSTALL_PREFIX=/usr/local ..
+make -j8
+$(PATH_TO_OPENCV)/build/bin/opencv_test_dnn --gtest_filter=*layers*
+```
+
+### Update Dawn's API
+
+Dawn is still under development: [implementation status](https://github.com/gpuweb/gpuweb/wiki/Implementation-Status). If Dawn’s API changes, we have to re-build Dawn and copy the `webgpu_cpp.cpp` file under Dawn’s **out/Release/gen/src/dawn** folder to the `webgpu_cpp.cpp` file in OpenCV’s **modules/dnn/src/webgpu/src** folder. Only copy the content in `namespce wgpu {}`, no need to change the `#include` header file.
```

Comment:
Could you please avoid copying `webgpu_cpp.cpp` into `modules/dnn/src/webgpu/src`? Instead you can add this file into `SOURCE` of `dnn` module when `WITH_WEBGPU` is set. Please look into `modules/dnn/CMakeLists.txt` and `ocv_glob_module_sources` for details. 

@alalek , any comments here?

Reply:
Done, I've moved this into `modules/dnn/CMakeLists.txt` and updated READEME.md. PTAL @huningxin @alalek !

Reply:
Looks good, thanks!